### DISCUSSION TITLE
I love the songs the ghost plays!

### DISCUSSION THREAD

**Discussion by manumanuma** [LouisSilva/LethalCompanyHarpGhost#7] [General]
Hi!
I've been watching youtube videos featuring this mod, and I love the songs the ghost plays.
I would love to know the names/titles.
Shazam hasn't been of much help.
Thanks beforehand!
Great mod!

**Comment by TexLeeV** [comment]
My girlfriend thought it sounded familiar so she spent half an hour trying to figure it out.  Apparently, it's called "The Dark Streets of Duskwall".  I attached a YouTube link as reference with a timestamp that is relative to what the ghost plays.

https://youtu.be/uulf97Tvv8I?t=84

I would suggest the mod acknowledges the song pieces as they are acknowledging other libraries that they are using in the source code.  Hope this helps

**Reply by manumanuma** [reply]
Wow thanks a lot!
What were the chances?
That's some nice hearing your gf has!

**Reply by wenbin97** [reply]
Much appreciated! Much love to you and your gf!

**Comment by LouisSilva** [comment]
I added the links to the music in the [Thunderstore readme](https://github.com/LouisSilva/LethalCompanyHauntedHarpist/blob/main/Thunderstore/README.md#music).
